test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,30 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // No arguments
+        assertNoneNull();
+
+        // One argument
+        assertHasNull((Object) null);
+        assertNoneNull("1");
+
+        // Multiple arguments, with null
+        assertHasNull(1, null, "Hello");
+        assertHasNull(null, null);
+
+        // Multiple arguments, without null
+        assertNoneNull("1", "");
+        assertNoneNull(0, new int[2]);
+    }
+
+    private void assertHasNull(Object... items) {
+        assertTrue(Utils.isAnyNull(items));
+    }
+
+    private void assertNoneNull(Object... items) {
```

Comment:
The function name sounds a bit weird. Maybe can go with assertNoNull or assertHasNoNull?